Libraries/react-native/react-native.js

```diff
@@ -60,11 +65,18 @@ var ReactNative = Object.assign(Object.create(require('React')), {
   // Plugins
   DeviceEventEmitter: require('RCTDeviceEventEmitter'),
   NativeModules: require('NativeModules'),
+  ReactIOSViewAttributes: require('ReactIOSViewAttributes'),
+  createReactIOSNativeComponentClass: require('createReactIOSNativeComponentClass'),
```

Comment:
I'm hacking on a way to automatically create the native components so we don't need to expose this or mirror validProps between JS and native, so hopefully we won't need this and most of the other exports here going forward.